packages/grid/x-data-grid/src/components/cell/index.ts

```diff
@@ -1,9 +1,21 @@
+import { IconButtonProps } from '@mui/material/IconButton';
+import { MenuItemProps } from '@mui/material/MenuItem';
+
+import {
+  GridActionsCellItem as InternalGridActionsCellItem,
+  GridActionsCellItemProps as InternalActionsCellItemProps,
+} from './GridActionsCellItem';
+
+export type GridActionsCellItemProps = InternalActionsCellItemProps<IconButtonProps, MenuItemProps>;
+export const GridActionsCellItem = InternalGridActionsCellItem as React.ForwardRefExoticComponent<
+  React.PropsWithoutRef<GridActionsCellItemProps> & React.RefAttributes<HTMLButtonElement>
+>;
+
```

Comment:
Here, the types of `GridActionsCellItem` is inserted by Material UI component props.